tests/test_asciiblock.py

```diff
@@ -22,44 +20,54 @@ def noop():
     pass
 
 
-def test_display():
-    scr_height = 40
-    scr_width = 100
-    fake_result = struct.pack('HHHH', scr_height, scr_width, 600, 616)
+try:
```

Comment:
instead of putting the whole test in try/catch block, do a check for ImportERror earlier and place it into if statement so we can reuse this logic for possible future tests.